### PR TITLE
Add Total Count Of Projects To Search Bar Placeholder

### PR DESCRIPTION
**Pull request by aravindparappil46**
This PR addresses Issue #10 

Screenshot of the change:
image

Please feel free to suggest different phrasing for the placeholder text, or any other feedback :) 

**Question:** This change has only been made to the `Projects` page. The placeholder in the `Contribute` page search box also says "Search for a project to contribute to...". Perhaps we need to reword that sentence to be more generic (since people might search for keywords in an issue as well?)

### CODE REVIEW

File: backend/apps/owasp/templates/search/project.html [hunk 1]
```diff
@@ -159,6 +159,11 @@ <h4 class="px-1">
         } = Vue;
         createApp({
             delimiters: ['${', '}'],
+            computed: {
+                searchPlaceholderText() {
+                    return `Search among ${this.projects.length} projects to contribute to...`;
```
Comment (arkid15r) [inline backend/apps/owasp/templates/search/project.html]: This is the right implementation. But it's a half-done -- we'd like to emphasize the total amount of projects OWASP has (somewhat around 235). So you could either hardcode that value or create another API endpoint and get this number from `Project::active_projects_count` (requires backend work and API endpoint naming thoughts)

**Review by arkid15r — COMMENTED** [review]
Thanks for the very first PR contribution into Nest code base.

**Comment by arkid15r** [comment]
> **Question:** This change has only been made to the `Projects` page. The placeholder in the `Contribute` page search box also says "Search for a project to contribute to...". Perhaps we need to reword that sentence to be more generic (since people might search for keywords in an issue as well?)

I see what's you're saying. Could you create an issue or maybe a PR with an alternative wording and take care of this?

Maybe "Find an issue to solve..."

File: backend/apps/owasp/templates/search/project.html (hunk 1 above)
Comment (aravindparappil46) [reply backend/apps/owasp/templates/search/project.html]: @arkid15r -- I have added an endpoint `/api/v1/owasp/search/project/active-projects-count` which just returns the `Project.active_projects_count()`. Figured this is also a "search" of sorts, so kept it in the existing `search.py`. 

Please let me know if the naming/location is not intuitive and more than happy to iterate on this :)

File: backend/apps/owasp/api/search/project.py [hunk 2]
```diff
@@ -36,3 +36,7 @@ def get_projects(query, attributes=None, limit=25):
 def projects(request):
     """Search projects API endpoint."""
     return JsonResponse(get_projects(request.GET.get("q", "")), safe=False)
+
+def active_projects_count(request):
```
Comment (arkid15r) [inline backend/apps/owasp/api/search/project.py]: Great idea. Do you think we could incorporate this data into the existing endpoint for projects? Basically, return a dict -- I think we can keep the API endpoints complexity lower this way.
```python
{"active_projects": Project.active_projects_count(), 
 "items": get_projects(request.GET.get("q", "")), safe=False
}```

**Review by arkid15r — COMMENTED** [review]
I like the way you addressed the problem with an additional endpoint for showing the up to date value instead of hard-coding it.

Could you address the suggestion below?

File: backend/apps/owasp/api/search/project.py (hunk 2 above)
Comment (aravindparappil46) [reply backend/apps/owasp/api/search/project.py]: Totally agree with this! Thanks @arkid15r !
Have pushed a commit :)

**Review by arkid15r — APPROVED** [review]
This is cool, I'm merging this in!

Thank you @aravindparappil46

**Pull request merged**